components/Enter/EnterPage.tsx

```diff
@@ -14,6 +16,13 @@ const EnterPage = () => {
   const contentRef = useRef()
   const buttonRef = useRef()
 
+  const currentTime = new Date().getTime()
+  const shipDay = "30 Jun 2023 09:00:00 UTC"
+  const alertEndDay = "2 Jul 2023 09:00:00 UTC"
+
+  const shouldOpenModal =
+    currentTime >= new Date(shipDay).getTime() && new Date(alertEndDay).getTime() >= currentTime
```

Comment:
niccceeeee 💯

Reply:
actually, this check makes it hard to test if changes are working.

can you remove the `startTime` so I can test now? I will not ship until popup should be active.

keep the `endTime` so it stops popup automatically